flink-cdc-common/src/main/java/com/ververica/cdc/common/event/ChangeEvent.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.common.event;
+
+/**
+ * Class {@code ChangeEvent} represents the change events of external systems, including {@link
+ * DataChangeEvent} and {@link SchemaChangeEvent}.
+ */
+public interface ChangeEvent extends Event {
```

Comment:
We can add `@PublicEvolving` annotation in 3.0 version, and then migrate to `@Public` once the API  become stable.